### DISCUSSION TITLE
Q: What approaches (or better, data!) have helped you make the case to colleagues/managers that there are tangible benefits which come along with contributing to OSS?

### DISCUSSION THREAD

**Discussion by iamh2o** [community/OSPO#18] [General]
*and benefits which go beyond 'FREE SOFTWARE!'*. :-)

So, I am a Scientist who can often pigeonhole myself as a Bioinformatician (where I do spend most of my time....).   The Bfx community is pretty `all in` w/r/t OSS, and I've managed to spend 15 years at companies which get significant benefit from OS Scientific s/w tools-   but I repeatedly have failed to convince even one of my managers to seriously put energy towards giving back.  I'm beginning a new gig, and already fear this pattern will continue.

{ Q.1 } But first, I ask myself... do the benefits outweigh the drawbacks?  What do you think?  And specifically what are some drawbacks?

I can only think of 2 worth mentioning; {{a}} As a company, if some staff work on OS projects, it will sometimes, but not always be directly relevant the company (tangent here,  fear of giving away innovation to competitors....  and {{ b }} fear of exposure to liability if the s/w causes harm in some way.
 
 
{ Q.2 } And what are the benefits which outweigh the above.  Observations are welcome, statement's backed by research / data extremely welcome.  I'll add my thoughts in a while , but I'll seed this section with a lonely bullet for now.

- ...

{ Q.3 } Lastly has anyone had similar experience's they'd like to share (particularly if you had a positive outcome.  Though cautionary tales have value as well.   

Thanks---
major

(edited by moderator for formatting only)

**Comment by king-jam** [comment]
Benefits definitely outweigh the drawbacks as long as you have trained developers who know not to step in any licensing minefields. The big drawback is Intellectual Property risks but benefit wise you can easily end up having a lot of influence in a project and/or not having to maintain functionality you added since a community may end up doing it for you.

I've worked within my company to help kickstart "Open Source Days" where approved projects get free contributions from the team. We looked at what OSS we were pulling in and made an effort to get pre-approval from the lawyers on the size and scope of changes. This then let us use it as an alternative to internal "hackathons" which helps provide some alternatives for devs and provides docs/bugfixes/enhancements to the projects we use internally. OSS has saved us $$$ millions so it's only reasonable we start giving back.

The big thing I've hit on is recruiting (open source - relevancy) and maintenance (why write something we own and maintain when we can take something and contribute back changes to a community).
